Sources/ProjectDescription/DeploymentTarget.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+// MARK: - DeploymentTarget
+
+public enum DeploymentTarget: Codable {
+    case iOS(String, [DeploymentDevice])
```

Comment:
`DeploymentDevice` is an `OptionSet` as such it gets array like syntax without needing to declare it as an array :)

`[.ipad, iphone]` == `.iphone | ipad`

Additionally it can mitigate the following funny case:

```swift
deploymentTarget: .iOS("12.2", [.ipad, .ipad, .ipad]),
```

Reply:
For the public facing api, as a suggestion, what do you think of adding labels?

`iOS(targetVersion: String, devices: DeploymentDevice)`